### DISCUSSION TITLE
New type of note in osu!taiko

### DISCUSSION THREAD

**Discussion by Flan-Fan** [ppy/osu#15374] [Ideas]
There maybe should be a new type of note where it is required to hit both sides of the drum at one (i.e if i had dfjk bindings i would hit it new type of note with d and f at the same time or j and k)

This may be pretty easy to implement, because you could potentially take the code that handles multiple inputs at the same time from mania and tweak it a bit for taiko

**Comment by bdach** [comment]
you mean like the large/"strong" hits? (see wiki: https://osu.ppy.sh/wiki/en/Game_mode/osu!taiko#taiko-notes)

that is already a thing.

**Reply by Flan-Fan** [reply]
No, I didn't mean that, I meant a note where you can hit both colours at the same time, not hit both notes of the same colour

**Reply by gagahpangeran** [reply]
You mean something like left/right don-kat?

![image](https://user-images.githubusercontent.com/4964985/139532363-21807964-2227-4b0f-8dd3-7f64a335481e.png)

![image](https://user-images.githubusercontent.com/4964985/139532391-dacab05f-6dad-4207-91b9-8a8de4b137f3.png)

I dont think there is no such note that represent both left/right don-kat in original taiko no tatsujin game. But I think it is interesting idea for new osu!taiko hit object. Not sure if there's demand from community for it.

**Reply by Flan-Fan** [reply]
Yeah I mean that
I just think it could be an interesting idea

**Reply by TheFearlessDeath500** [reply]
> Yeah I mean that
> I just think it could be an interesting idea

This is already present in a similar Taiko simulator named OpenTaiko. While it does not support OSU files, it still functions and adds more core features to Taiko gameplay.
